src/client/components/voyage/InputField.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+
+class InputField extends Component {
+  render() {
+    const { label, name, step, unit, value, onDataInputChange } = this.props;
+    return (
+      <label>
+        {label}
+        <input
+          type="number"
+          name={name}
+          step={step}
+          value={value}
+          placeholder={`0 ${unit}`}
```

Comment:
This is not the "proper" way to do default props in react.

What you should do is to make an actual prop so you can set the placeholder value when making the component, but if it the prop is left empty, render whatever is in your defaultProps object. The placeholder prop should of course be optional - otherwise it does no make sense to declare defaults :)

Reply:
Re-configured default props properly for Input Field Number component.